### ISSUE TITLE
multiple AND-combinated `validate.pattern` clause

### ISSUE THREAD

**Issue by drzraf** [getgrav/grav-plugin-form#310]
It's sometimes easier to provide multiple patterns for a given field rather than trying to put all several constraints into one regex.
Example: test that a value is an actual URL and contains at least a set of handful fields.

**Comment by mahagr** [comment]
FYI: Pattern is also used by the browser, which may complicate doing this.

**Comment by drzraf** [comment]
Agreed. That would probably imply the use of external javascript for custom validity in order to make it more programmatic. Some libs like http://parsleyjs.org/ seem to do this.
